bciers/apps/administration/tests/components/operations/OperationInformationForm.test.tsx

```diff
@@ -584,4 +596,153 @@ describe("the OperationInformationForm component", () => {
+      container.querySelector("#root_section3_registration_purposes"),
+    ).toHaveTextContent("New Entrant Operation");
+    expect(
+      container.querySelector("#root_section3_new_entrant_preface"),
+    ).toHaveTextContent("New Entrant Operation");
+    expect(
+      screen.getByText(/when is this operation's date of first shipment\?/i),
+    ).toBeVisible();
+    expect(screen.getByText(/on or before march 31, 2024/i)).toBeVisible();
+    expect(
+      screen.getByText(/new entrant application and statutory declaration/i),
+    ).toBeVisible();
+    expect(screen.getByText("testpdf.pdf")).toBeVisible();
+    expect(
+      screen.getByRole("link", {
+        name: /preview/i,
+      }),
+    ).toHaveAttribute("href", mockDataUri);
+  });
+
+  it("should edit and submit the new entrant application form", async () => {
+    useSession.mockReturnValue({
+      data: {
+        user: {
+          app_role: "industry_user_admin",
+        },
+      },
+    });
+    const testSchemaWithNewEntrant: RJSFSchema = {
+      type: "object",
+      properties: {
+        section1: {
+          title: "Section 1",
+          type: "object",
+          properties: {
+            name: {
+              type: "string",
+              title: "Operation Name",
+            },
+          },
+        },
+        section2: {
+          title: "Section 2",
+          type: "object",
+          properties: {
+            type: {
+              type: "string",
+              title: "Operation Type",
+            },
+          },
+        },
+        section3: {
+          title: "Registration Information",
+          type: "object",
+          properties: {
+            registration_purposes: {
+              type: "array",
+              items: {
+                type: "string",
+                enum: ["New Entrant Operation"],
+              },
+            },
+            new_entrant_preface: {
+              type: "string",
+            },
+            date_of_first_shipment: {
+              type: "string",
+              enum: [
+                "On or before March 31, 2024",
+                "On or after April 1, 2024",
+              ],
+            },
+            new_entrant_application: {
+              type: "string",
+              title: "New Entrant Application and Statutory Declaration",
+              format: "data-url",
+            },
+          },
+        },
+      },
+    };
+    fetchFormEnums();
+    getOperationWithDocuments.mockResolvedValueOnce(newEntrantFormData);
+
+    render(
+      <OperationInformationForm
+        formData={newEntrantFormData}
+        schema={testSchemaWithNewEntrant}
+        operationId={operationId}
+      />,
+    );
+
+    await userEvent.click(screen.getByRole("button", { name: "Edit" }));
+    const afterAprilRadioButton = screen.getByLabelText(
+      "On or after April 1, 2024",
+    );
+
+    await userEvent.click(afterAprilRadioButton);
+    const mockFile = new File(["test"], "test.pdf", {
```

Comment:
The mockDataUri is also called `test.pdf`, so I don't know that this will be able to confirm the new doc is uploaded